jms/1.7/modules/ROOT/pages/jms-connector-reference.adoc

```diff
@@ -577,14 +577,14 @@ The maximum number of times a message can be redelivered and processed unsuccess
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Jmsx User ID a| String |  |  |
-| Jmsx App ID a| String |  |  |
-| Jmsx Delivery Count a| Number |  |  |
-| Jmsx Group ID a| String |  |  |
-| Jmsx Group Seq a| Number |  |  |
-| Jmsx Producer TXID a| String |  |  |
-| Jmsx Consumer TXID a| String |  |  |
-| Jmsx Rcv Timestamp a| Number |  |  |
+| Jmsx User ID a| String | The user identity that sends the message. |  |
+| Jmsx App ID a| String | The application identity that sends the message |  |
```

Comment:
```suggestion
| Jmsx App ID a| String | Identity of the application that sends the message. |  |
```

Reply:
I'm suggesting these changes because it's not the identity that sends the message, and it's not the transaction ID that produces or consumes. Identities don't do the action.